### PR TITLE
Added season 2018

### PR DESCRIPTION
**Pull request by jeuxjeux20**
GetRecentMatches couldn't work without it soo....

### CODE REVIEW

File: RiotSharp/Endpoints/MatchEndpoint/Enums/Season.cs [hunk 1]
```diff
@@ -57,7 +57,17 @@ public enum Season
...
+        
+        /// <summary>
+        /// Pre season 2018.
+        /// </summary>
+        PreSeason2018 = 10,
+        
+        /// <summary>
+        /// Season 2018.
+        /// </summary>
+        Season2017 = 11,
```
Comment (JanOuborny) [inline RiotSharp/Endpoints/MatchEndpoint/Enums/Season.cs]: Please remove the trailing coma

**Comment by JanOuborny** [comment]
@BenFradet You might want to release a new version to fix this.

**Comment by jeuxjeux20** [comment]
fixed :)

**Comment by jeuxjeux20** [comment]
Is there still some branch or commit for latest nuget version of the library ?

**Comment by JanOuborny** [comment]
@BenFradet If you want a NuGet package of the latest version, follow the instructions in the readme file.

**Pull request merged**